dex-auth.go

```diff
@@ -26,7 +26,12 @@ func (cluster *Cluster) oauth2Config(scopes []string) *oauth2.Config {
 }
 
 func (config *Config) handleIndex(w http.ResponseWriter, r *http.Request) {
-	renderIndex(w, config)
+
+	if len(config.Clusters) == 1 && r.URL.String() == config.Web_Path_Prefix {
```

Comment:
I'm just going by the names of things, but should the second condition test that the URL starts with the `config.Web_Path_Prefix` taking into account how URL prefixes are different to string prefixes?

Reply:
the second condition is required because the liveness/readiness probe is configured to be /healthz which is not exposed by the app. so in case Web_Path_Prefix == "/", all the /healthz requests will trigger a redirect.